mkdocs.yml

```diff
@@ -66,6 +65,8 @@ markdown_extensions:
       emoji_generator: !!python/name:material.extensions.emoji.to_svg
   - pymdownx.tasklist:
       custom_checkbox: true
+  - toc:
+      permalink: true
```

Comment:
do we want to use the paragraph symbol or something like `#` instead?

Reply:
up to you, i'll leave it as is and we can always change it. i just took it from pydantic cuz i saw they had it and we didnt